components/Spinner/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from "./spinner.module.css";
+
+export const Spinner: React.FC = () => {
+  return (
+    <div className={styles["lds-ellipsis"]}>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  );
+};
+
+export const LoadingPage: React.FC<{ text?: string }> = ({
+  text = "Wait a bit, Tina is loading data from GitHub",
```

Comment:
I'm assuming this shows when you are working locally, as well?  Do we just want to truncate the "from GitHub" to make it agnostic?

"Wait a bit, Tina is loading data"